control-plane/tests/test_service_persistence_restore_readiness.py

```
@@ -79,6 +79,29 @@ def test_service_wires_restore_readiness_internal_collaborators(self) -> None:
             "RestoreReadinessService should delegate readiness projection to a dedicated collaborator",
         )
 
+    def test_service_decomposes_restore_validation_into_internal_helpers(
+        self,
+    ) -> None:
+        store, _ = make_store()
```

Ruff (0.15.12)

[error] Undefined name `make_store`

(F821)